src/analysis/analytic/basic.lean

```diff
@@ -632,12 +632,25 @@ begin
         apply norm_tsum_le_tsum_norm,
         convert p.change_origin_summable_aux3 k h,
         ext a,
-        tidy
+        rcases a with ⟨a1, ⟨a2, ha2⟩⟩,
+        refl
       end
   ... = tsum (λ i, ∥(p i.1).restr i.2.1 i.2.2 x∥ * ↑r ^ k :
     (Σ (n : ℕ), {s : finset (fin n) // finset.card s = k}) → ℝ) :
-      by { rw tsum_mul_right, convert p.change_origin_summable_aux3 k h, tidy }
-  ... = tsum (A ∘ change_origin_summable_aux_j k) : by { congr, tidy }
+      begin
+        rw tsum_mul_right,
+        convert p.change_origin_summable_aux3 k h,
+        ext a,
+        rcases a with ⟨a1, ⟨a2, ha2⟩⟩,
```

Comment:
Here and above / below:
```suggestion
        ext ⟨a1, ⟨a2, ha2⟩⟩,
```
You can use `rcases` patterns directly with `ext`.

Reply:
@robertylewis already gave me this advice on #5507, but I hadn't merged it into this yet. Thank you both for this trick.